src/legacy/core_plugins/metrics/public/visualizations/components/top_n.js

```diff
@@ -59,14 +60,21 @@ class TopN extends Component {
           onClick={this.handleClick({ lastValue, ...item })}
           style={styles.row}
         >
-          <td width="1*" className="tvbVisTopN__label">{ item.label }</td>
+          <td width={`${labelWidth}%`} className="tvbVisTopN__label">
+            <EuiToolTip
+              position="right"
+              content={item.label}
+            >
+              <span>{ item.label }</span>
```

Comment:
Instead of wrapping in a tooltip that will apply to all labels no matter if they're truncated or not, can you just supply a `title` attribute to the `<span>`?